src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -132,12 +132,16 @@ public function overload(string $path, string ...$extraPaths): void
     public function populate(array $values, bool $overrideExistingVars = false): void
     {
         $updateLoadedVars = false;
+        $alwaysSetEnvSuperglobal = (false === strpos(ini_get('variables_order'), 'E')) ? true : false;
```

Comment:
$alwaysSetEnvSuperGlobal maybe ?

Reply:
I took over the spelling from https://github.com/symfony/symfony/blame/0375b4aed9eb00901222dfcc53112786bb9fcfad/src/Symfony/Component/Dotenv/Tests/DotenvTest.php#L341. If you want then I can change all of them, but I don't think it's needed here...